code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -659,6 +659,77 @@ func (ssc *StorageSmartContract) GetWriteMarkersHandler(ctx context.Context,
 
 }
 
+func (ssc *StorageSmartContract) GetWritenAmountHandler(
```

Comment:
This method should have `allocation_id` as input, otherwise it would return data size of all allocation, I doubt if this is needed. So as the GetReadAmountHandler() below

Reply:
done

Reply:
Fix the typo of *Writen* so should be GetWrittenAmountHandler